TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/logic/clusters.py

```diff
@@ -1,8 +1,46 @@
-from console_link.models.cluster import Cluster
+from console_link.models.cluster import Cluster, HttpMethod
+import logging
+
+logger = logging.getLogger(__name__)
 
 
 def cat_indices(cluster: Cluster, as_json=False):
-    as_json_suffix = "?format=json" if as_json else ""
+    as_json_suffix = "?format=json" if as_json else "?v"
     cat_indices_path = f"/_cat/indices{as_json_suffix}"
     r = cluster.call_api(cat_indices_path)
     return r.json() if as_json else r.content
+
+
+def connection_check(cluster: Cluster):
+    cluster_details_path = "/"
+    caught_exception = None
+    r = None
+    try:
+        r = cluster.call_api(cluster_details_path, timeout=3)
+    except Exception as e:
+        caught_exception = e
+        logging.debug(f"Unable to access cluster: {cluster} with exception: {e}")
+    access_result = {}
+    if caught_exception is None:
+        response_json = r.json()
+        access_result['connection_message'] = "Successfully connected!"
+        access_result['cluster_version'] = response_json['version']['number']
+        access_result['connection_established'] = True
```

Comment:
might be nice to make this access_result something like a named tuple or a very simple class so we can make a type guarantee on the return and the caller knows exactly what to assume they can get